### DISCUSSION TITLE
Use script variant in Jetbrains IDEs

### DISCUSSION THREAD

**Discussion by repoles** [MonoLisaFont/feedback#189] [Q&A] [answered]
Hi there 👋 

Is it possible to enable MonoLista script variant inside the Jetbrains IDEs? How?

**Comment by okonet** [comment, marked as answer]
It’s not possible to enable it via OpenType feature. That’s why we build the customization tool. There you can choose to enable script by default and download the custom font which you can then use in the JetBrains products or any other app. Go to /orders and check it out. If you need more help, let me know.

**Comment by kimpenhaus** [comment]
hey @okonet,

not sure if I have a misunderstanding on how the script variant should work or I have used the customization tool wrong... maybe you can give me some hints? :)

Ia m using JetBrains RIDER ide and my assumtion was/is that the keywords are shown in the script variant but as you can see that's not the case:

<img width="2122" alt="grafik" src="https://github.com/user-attachments/assets/19139af8-42aa-4ae3-8624-30781c99f4e3">

My customization looks as follows (see the JB suffix):

<img width="539" alt="grafik" src="https://github.com/user-attachments/assets/ddef438e-a55c-415d-816d-024eab6bd79d">

and the editor font set to the JB suffixed font:

<img width="826" alt="grafik" src="https://github.com/user-attachments/assets/b3193ebc-4a8a-44d5-84dc-2ca7824ee21e">

Is there anything wrong with my setup/configuration or is it just my assumption that was/is wrong?

thanks for helping

**Reply by bebraw** [reply]
Script variant applies only to italics.

Based on your screenshot you might have to adjust your IDE to use italics where you want to use the script variant. I imagine there must be some way to achieve this in JetBrains as they seem to use regular weight for most characters in your example. Essentially you should find a way to map a keyword type to the italic version of the font to achieve the result you want.

**Reply by bebraw** [reply]
Check https://www.jetbrains.com/help/rider/Configuring_Colors_and_Fonts.html#customize-color-scheme .

![IMG_0312](https://github.com/user-attachments/assets/3be2e8c8-d843-47c9-a718-f92c5fea2a29)

It's that "Italic" checkbox that I mean.

**Reply by kimpenhaus** [reply]
hey @bebraw - thanks for the quick reply. understood and will figure out how to customize to my needs. thanks!
